modules/core/src/main/java/org/apache/ignite/internal/util/OffheapReadWriteLock.java

```diff
@@ -34,13 +35,27 @@
  * </pre>
  */
 public class OffheapReadWriteLock {
+    /** @see #IGNITE_OFFHEAP_RWLOCK_SPIN_COUNT */
+    public static final int DFLT_OFFHEAP_RWLOCK_SPIN_COUNT = 32;
+
+    /** */
+    @SystemProperty(value = "Count spin lock iterations before fallback to block", type = Long.class,
```

Comment:
A number of spin-lock iterations to take before falling back to the blocking approach.